evm-contracts/src/v0.7/dev/OperatorGenerator.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.7.0;
+
+import "./Operator.sol";
+
+/**
+ * @title Operator Generator
+ * @notice Generates Operator contracts for node operators
+ */
+contract OperatorGenerator {
+
+    address public link;
+
+    event OperatorCreated(address indexed operator, address indexed owner);
+
+    /**
+     * @param linkAddress address
+     */
+    constructor(address linkAddress) public {
+        link = linkAddress;
+    }
+
+    /**
+     * @notice Create a new Operator contract with the msg.sender as owner
+     * @return operatorAddress
+     */
+    function createOperator() external returns (address operatorAddress){
```

Comment:
How about we make this the fallback function so that any transaction sent here will generate a contract? Seems like it will be even easier for people.